### ISSUE TITLE
Request URI too large

### ISSUE THREAD

**Issue by goldhoorn** [mikaa123/umlify#16]
When i try to run the application everythings seems to work fine, but the resulting png contains the following (text) and is therefore invalid:

```
  1 <html>
  2 <head><title>414 Request-URI Too Large</title></head>
  3 <body bgcolor="white">
  4 <center><h1>414 Request-URI Too Large</h1></center>
  5 <hr><center>nginx/1.6.2</center>
  6 </body>
  7 </html>
```

**Comment by mikaa123** [comment]
Hey there,
Thanks for reporting this problem!

It seems like the project you are trying to convert into a diagram is a little too big. The PNG is generated from a REST call to a URI that takes the diagram information as querystring parameters. However, the resulting URI is too big.

Unfortunately, I don't see any way to bypass this limitation.

**Comment by goldhoorn** [comment]
I was able to generate a graph:
i added in lib/umlify/runner.rb ( umlify-1.2.6)

```
 64       File.open('debug.text', 'wb') do |file|
 65         file << @diagram.get_uri
 66       end
```

I pasted the content of debug.txt in http://yuml.me/diagram/scruffy/class/draw and this works for me as a workaround. Maybe this debug-file-generation could be added if the normal way fails...

**Comment by ailjushkin** [comment]
I think rest request should be sent as POST... Not see it as a problem to refactor, before this refactoring works, this gem could not be used for large projects...

**Comment by mikaa123** [comment]
Yes that's true, but the underlying service only takes GET requests...
